### PR TITLE
use six.iteritems

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

**Review by emord — APPROVED** [review]

File: corehq/apps/app_manager/views/app_summary.py [hunk 1]
```diff
@@ -142,7 +143,7 @@ def _translate_name(names, language):
     try:
         return names[language]
     except KeyError:
-        first_name = names.iteritems().next()
+        first_name = six.iteritems(names)
```
Comment (calellowitz) [inline corehq/apps/app_manager/views/app_summary.py]: this looks different than what was there before. i think you are missing `.next()`

File: corehq/util/pagination.py [hunk 2]
```diff
@@ -195,7 +196,7 @@ def __iter__(self):
 
         retried = {}
         while self.state.retry != retried:
-            for item_id, retries in list(self.state.retry.iteritems()):
+            for item_id, retries in list(six.iteritems(self.state.retry)):
```
Comment (calellowitz) [inline corehq/util/pagination.py]: why does this need to be wrapped in `list`

**Review by calellowitz — CHANGES_REQUESTED** [review]
just a few minor questions. there were also few files where you did not `import six`, assuming thats because those were already there but did not check each one

File: corehq/apps/app_manager/views/app_summary.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/app_manager/views/app_summary.py]: Good catch - updated

File: corehq/util/pagination.py (hunk 2 above)
Comment (nickpell) [reply corehq/util/pagination.py]: ```list``` looks unnecessary to me.

@snopoke was there a reason ```list``` was used here?

Comment (snopoke) [reply corehq/util/pagination.py]: Can't think why I'd have added the `list`. Doesn't seem necessary.

**Pull request merged**